src/standalone/ImageBoxControl/ImageBox.tsx

```diff
@@ -0,0 +1,158 @@
+import React from "react";
+import { Grid, Link } from "@material-ui/core";
+import { IDownscaleProps } from "../../utils/processImage";
+import { makeStyles } from "@material-ui/core/styles";
+import GroupBox from "../GroupBox";
+
+import { useDialogContext } from "../../framework";
+import { showImageDialog } from "../../non-standalone/Dialog";
+export type ImageBoxInputElement = { name: string; value: string };
+
+/**
+ * Object properties for image
+ */
+export interface ImageSelectorBoxProps {
+	/**
+	 * The base64 string of image
+	 */
+	src: string;
+	/**
+	 * Boolean flag to set primary image
+	 */
+	setPrimary: boolean;
+}
+export interface ImageBoxProps {
```

Comment:
extend the interface instead of copy pasting the fields. `export interface ImageBoxProps extends IDialogConfigImageBox`